.changeset/friendly-llamas-speak.md

```diff
@@ -0,0 +1,9 @@
+---
+"@salt-ds/lab": minor
+---
+
+Added `visibleMonths` to date picker, it renders a single calendar for a range selection. It's recommended using it when the space is limited.
```

Comment:
this change description is confusing. It comes across as … if you are a range you get a single calendar ?